### PR TITLE
feature/history_page_change_map: implement same as on search page

### PR DESCRIPTION
**Pull request by K0sen**

### CODE REVIEW

File: app/Actions/Place/Checkin/GetUserCheckInCollectionPresenter.php [hunk 1]
```diff
@@ -30,15 +30,22 @@ public static function presentCollectionItem(Checkin $checkIn, float $rating)
...
-                'id'        => $checkIn->place->id,
-                'latitude'  => $checkIn->place->latitude,
-                'longitude' => $checkIn->place->longitude,
-                'zip'       => $checkIn->place->zip,
-                'address'   => $checkIn->place->address,
-                'city'      => $checkIn->place->city->name,
-                'category'  => $checkIn->place->category->name,
-                'createdAt' => $checkIn->place->created_at->toDateTimeString(),
-                'name'      => $checkIn->place->localization[0]->place_name,
+                'id'           => $checkIn->place->id,
```
Comment (nemoipaha) [inline app/Actions/Place/Checkin/GetUserCheckInCollectionPresenter.php]: We have PlacePresenter class for that

File: frontend/src/components/place/PlaceVisitedPreview.vue [hunk 2]
```diff
@@ -15,7 +15,7 @@
                 </div>
                 <h2 class="title">
                     <router-link :to="`/places/${checkIn.id}`">
-                        {{ checkInPlace.name }}
+                        {{ checkInPlace.localization[0].name }}
```
Comment (nemoipaha) [inline frontend/src/components/place/PlaceVisitedPreview.vue]: create computed prop for that

File: frontend/src/store/modules/user/history/getters.js [hunk 3]
```diff
@@ -11,4 +11,24 @@ export default {
         let placeId = state.checkIns.byId[id].placeId;
         return state.places.byId[placeId];
     },
+
+    getMapboxCenter: () => (placeItems, placeIds) => {
```
Comment (nemoipaha) [inline frontend/src/store/modules/user/history/getters.js]: This method canbe inside `search` module, also with `getMapboxToken`

**Comment by K0sen** [comment]
Corrected

**Pull request merged**